user-guide/Getting_started/Installing_a_DMA/Installing_DM_using_the_DM_Installer.md

```diff
@@ -33,128 +33,41 @@ The DataMiner installer allows you to run a default DataMiner installation, whic
 
-### Default DataMiner installation
+## DataMiner installation
 
-1. Make sure you are logged into Windows with the correct user account:
-
-   - If you are using the DataMiner 10.2 installer or newer, use an account with administrator rights. During the installation, the user account will automatically be added to a local DataMiner user group named *Administrator (installer)*.
-   - If you are using a DataMiner installer older than version 10.2, use the server's local Administrator account. Do not use a regular user account with administrative rights.
+1. Make sure you are logged into Windows with the server's local Administrator account. Do not use a regular user account with administrative rights.
 
 1. Double-click the setup executable.
 
 1. Click *Install*.
 
-1. Enter the DataMiner ID.
-
-   > [!IMPORTANT]
-   > To get this DataMiner ID, contact <dataminer.licensing@skyline.be>. The DataMiner ID will uniquely identify the DataMiner Agent you are installing.
-   >
-   > If you are a Skyline employee, use the procedure from the [Skyline internal documentation](https://internaldocs.skyline.be/Corporate/OperatingProcedures/OP_RequestDecommissionMaintainInternalServer/Requesting_a_DMA_ID_or_a_DataMiner_license_for_an_internal_DataMiner_Agent.html) instead.
-
-1. Click next.
-
    The progress of the installation will be displayed. A *cancel* button in the lower right corner allows you to cancel the installation process if necessary.
 
-1. Once the installation is complete, click *next*.
-
-1. Click *go to Request.lic* to browse to *Request.lic*.
-
-1. Send the *Request.lic* file to <dataminer.licensing@skyline.be>, and wait until you receive the reply.
-
-   > [!TIP]
-   > See also: [Obtaining a DataMiner license](xref:DataminerLicenses)
-
-   > [!NOTE]
-   > If you are a Skyline employee, use the procedure from the [Skyline internal documentation](https://internaldocs.skyline.be/Corporate/OperatingProcedures/OP_RequestDecommissionMaintainInternalServer/Requesting_a_DMA_ID_or_a_DataMiner_license_for_an_internal_DataMiner_Agent.html) instead.
-
-1. Once you have received the license files, save these somewhere on the computer; however, not in the "Skyline DataMiner" folder.
+   Once the installation is complete, the configuration window will be displayed.
 
-1. In the *License* tab of the DataMiner Installer, click *browse and upload*, and navigate to the license files.
+1. Click *Start*.
 
-1. Once all files have been uploaded successfully, click *restart DataMiner*.
+1. Select the database type.
 
-1. When DataMiner has successfully restarted, click *Close*.
+   - If you selected Self-hosted - External Storage please check [configure the databases](xref:Configuring_dedicated_clustered_storage).
 
-At this point, the basic installation is complete. However, to be able to make full use of all available DataMiner features, you will also need to:
+1. Enter the DataMiner Info.
```

Comment:
This is a bit vague. Could you make sure it's clear what exactly should be filled in in which field(s)?